CMakeLists.txt

```diff
@@ -25,11 +25,6 @@ endif()
 
 include(${PROJECT_SOURCE_DIR}/toolchains/utils.cmake)
 
-# cmake output configuration
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
Yeah CMake is a little annoying for outputs on desktop platforms, everything would at the same level (libraries and executable with resources), but it's ok to remove it, just makes things more clear on what is the executable (thinking of linux and raspberry pi mainly).